src/cfn/template.yaml

```diff
@@ -453,6 +435,57 @@ Resources:
 
   # Computing & Containers
 
+  AWSSDKLayer:
+    Type: AWS::Serverless::LayerVersion
+    Properties:
+      LayerName: MediaAnalysisAWSSDK
+      Description: Latest confirmed compatible AWS SDK
+      ContentUri: ../backend/functions/layers/aws_sdk/
+      CompatibleRuntimes:
+        - nodejs8.10
+        - nodejs10.x
+
+  FPLayer:
+    Type: AWS::Serverless::LayerVersion
+    Properties:
+      LayerName: MediaAnalysisFP
+      Description: Layer containing functional programming libs
+      ContentUri: ../backend/functions/layers/fp/
+      CompatibleRuntimes:
+        - nodejs8.10
+        - nodejs10.x
+
+  Orchestrator:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: index.handler
+      Runtime: nodejs10.x
+      CodeUri: ../backend/functions/orchestrator/dist/
+      Timeout: 10
+      MemorySize: 512
+      Layers:
+        - !Ref AWSSDKLayer
+        - !Ref FPLayer
+      Policies:
+        - AmazonDynamoDBFullAccess
+        - AmazonECS_FullAccess
+        - AWSLambdaBasicExecutionRole
+        - AWSLambdaInvocation-DynamoDB
+      Environment:
+        Variables:
+          CLUSTER: !Ref ECSCluster
+          RETRY_THRESHOLD: !Ref RetryThreshold
+          SUBNETS: !Sub ${PrivateSubnet1},${PrivateSubnet2},${PrivateSubnet3}
+          TASKS_TABLE_NAME: !Ref TasksDynamoTable
+          TASK_NAME: transcriber
```

Comment:
Can we get this as reference from the `TransciberTaskDefinition`?

Reply:
unfortunately not - AWS::ECS::TaskDefinition returns the ARN. but this is hardcoded in the template a bit too much for my liking - perhaps we could parameterise it then default to `transcriber`?

Reply:
Yeah I think that would be a good idea